build/source/driver/summa_init.f90

```diff
@@ -65,7 +65,7 @@ subroutine summa_initialize(summa1_struc, err, message)
  USE summa_type, only:summa1_type_dec                        ! master summa data type
  ! subroutines and functions: initial priming
  USE summa_util, only:getCommandArguments                    ! process command line arguments
- USE summaFileManager,only:summa_SetDirsUndPhiles            ! sets directories and filenames
+ USE summaFileManager,only:summa_SetTimesDirsUndPhiles       ! sets directories and filenames
```

Comment:
Let's rename this now while it's being changed - `summa_SetTimesDirsAndFiles`

Reply:
Done also renamed `Philes` to `Files`